src/box/errcode.h

```diff
@@ -295,6 +295,7 @@ struct errcode_record {
 	/*240 */_(ER_COMPLEX_FOREIGN_KEY_FAILED, "Foreign key constraint '%s' failed: %s") \
 	/*241 */_(ER_WRONG_SPACE_UPGRADE_OPTIONS, "Wrong space upgrade options: %s") \
 	/*242 */_(ER_NO_ELECTION_QUORUM,	"Not enough peers connected to start elections: %d out of minimal required %d")\
+	/*243 */_(ER_SSL,			"%s") \
```

Comment:
Why do you need the new code? I don't see it used in this patch neither in the EE patch.

Reply:
Applier uses `box_error_code()` for better logging. In particular, it [logs](https://github.com/tarantool/tarantool/blob/73733e51331027424ddce117c09402b7d61db490/src/box/applier.cc#L115-L133) "will retry every X seconds" only on retryable error while `ER_PROC_LUA`, which is currently returned by `box_error_code()` for `SSLError`, isn't retryable. I moved the code adding the new error code to a separate commit and added a test case to `unit/error.test` checking `box_error_code()`.